README.md

```diff
@@ -386,73 +386,40 @@ Try to match the expression. If the match does not succeed, just return
 
 #### & { _predicate_ }
 
-The predicate is a piece of JavaScript code that is executed as if it was inside
-a function. It gets the match results of labeled expressions in preceding
-expression as its arguments. It should return some JavaScript value using the
-`return` statement. If the returned value evaluates to `true` in boolean
-context, just return `undefined` and do not consume any input; otherwise
-consider the match failed.
-
-The code inside the predicate can access all variables and functions defined in
-the initializer at the beginning of the grammar.
+This is a positive assertion. No input is consumed.
 
-The code inside the predicate can also access location information using the
-`location` function. It returns an object like this:
+The predicate should be JavaScript code, and it's executed as a
+function. Curly braces in the predicate must be balanced.
 
-```javascript
-{
-  source: options.grammarSource,
-  start: { offset: 23, line: 5, column: 6 },
-  end: { offset: 23, line: 5, column: 6 }
-}
-```
+The predicate should `return` a boolean value. If the result is
+truthy, the match result is `undefined`, otherwise the match is
```

Comment:
change `the` to `it's` to make it clear that the match result is for the predicate, not the expression it applies to?

Reply:
Done